client/src/Components/Questions.js

```diff
@@ -50,6 +52,17 @@ const Questions = (props) => {
 		});
 		setIsChecked(false);
 	};
+	const submitHandler = (e) => {
+		e.preventDefault();
+
+		if(answer.value=="") {
+			setModalText("Are you sure you want to skip this question?");
+
```

Comment:
you can use 
if(!answer.value){

Reply:
well "" is default in default state and that works for me

Reply:
i added some style in next pull, but bootstrap does some unexpected staff as well. i'll try to cnter it later and it's easier to do as all the messages are same component and 1 css file. I can go on with form. but the point of this ticket was fixing required field and keeping message at the same time. I think Sabia has a ticket on styling this component